packages/suite/src/utils/wallet/transactionUtils.ts

```diff
@@ -76,11 +75,10 @@ export const sumTransactionsFiat = (
 ) => {
     let totalAmount = new BigNumber(0);
     transactions.forEach(tx => {
-        // count only recv/sent txs
-        if (tx.type === 'sent') {
-            totalAmount = totalAmount
-                .minus(toFiatCurrency(tx.amount, fiatCurrency, tx.rates, -1) ?? 0)
-                .minus(toFiatCurrency(tx.fee, fiatCurrency, tx.rates, -1) ?? 0);
+        if (tx.type === 'sent' || tx.type === 'self') {
+            totalAmount = totalAmount.minus(
+                toFiatCurrency(tx.amount, fiatCurrency, tx.rates, -1) ?? 0,
```

Comment:
there is a dilema if we have 2 tx: one sent, one recv. each with amount 0.1 and fee 0.01, then what we show in txs overview is
\- 0.1
\+ 0.1
(cause we don't display fee in txs list)

which would add to 0, but in reality you lost more than received due to fee paid on sent tx. so shouldn't the sum dor the whole day show you that at the end of the day you had 0.01 less? imo yes, but then everyone will be confused because all they see is 
\- 0.1
\+ 0.1
which makes sense should lead to 0 delta at the end of the day, not -0.01.

So for now I would keep it this way and let's see if someone will find it more confusing than it was before or less :)

Reply:
cc @matejzak